iroh-cli/tests/cli.rs

```diff
@@ -482,7 +474,6 @@ async fn cli_provide_persistence() -> anyhow::Result<()> {
 }
 
 #[test]
-#[ignore = "flaky"]
 fn cli_provide_addresses() -> Result<()> {
```

Comment:
I've seen this fail on at least July 9th: https://github.com/n0-computer/iroh/actions/runs/9886682346/job/27306897440